common/metrics/service_wrapper.go

```diff
@@ -45,24 +45,24 @@ type (
 )
 
 const (
-	scopeNameDeprecateDomain                = "DeprecateDomain"
```

Comment:
Can we create a subscope then? Once we get the users scope, create a subscope with the cadence name and use that everywhere.

At the very least, this should be a constant that  all metrics concatenate instead of pasting the same string everywhere.